README.md

```diff
@@ -34,3 +39,13 @@ If you use this software in a scientific publication, please consider citing one
 
 - Cole Brokamp, Chris Wolfe, Todd Lingren, John Harley, Patrick Ryan. Decentralized and Reproducible Geocoding and Characterization of Community and Environmental Exposures for Multi-Site Studies. *Journal of American Medical Informatics Association*. 25(3). 309-314. 2018. [*Download*](https://colebrokamp-website.s3.amazonaws.com/publications/Brokamp_JAMIA_2017.pdf).
 - Cole Brokamp. DeGAUSS: Decentralized Geomarker Assessment for Multi-Site Studies. *Journal of Open Source Software*. 2018. [*Download*](https://colebrokamp-website.s3.amazonaws.com/publications/Brokamp_JOSS_2018.pdf).
+
+### Supported By:
```

Comment:
Let's make this and the "Used and Trusted By" level 2 headings (`##` instead of `###`) and remove the `:` to make them stylistically similar to the other headings.